src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Parent class of address,email,phone
+ */
```

Comment:
The comment shouldn't be reverse engineered. Please try to describe the class for what it is, instead of defining it as a parent class.